src/test/java/io/crate/consumer/CrateConsumerConstants.java

```diff
@@ -0,0 +1,36 @@
+package io.crate.consumer;
+
+public class CrateConsumerConstants {
+
+    private static final String TABLE_NAME = "benchmark.history";
+
+    static final String CRATE_HOST = "jub.crate.host";
+    static final String CRATE_HTTP_PORT = "jub.crate.http";
+    static final String CRATE_TRANSPORT_PORT = "jub.crate.transport";
+
+    static final String CREATE_TABLE_IF_NOT_EXISTS_STMT = String.format("CREATE TABLE IF NOT EXISTS %s (" +
+            " benchmark_group STRING PRIMARY KEY," +
```

Comment:
setting the primary keys like this makes it impossible to store multiple runs against the same version, is that really okay?